pkg/handlers/gitops.go

```diff
@@ -154,6 +155,18 @@ func (h *Handler) InitGitOpsConnection(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// If a branch is not provided, use the default branch
+	if downstreamGitOps.Branch == "" {
+		err := gitops.UpdateDownstreamGitOps(a.ID, d.ClusterID, downstreamGitOps.RepoURI, defaultBranchName,
+			downstreamGitOps.Path, downstreamGitOps.Format, downstreamGitOps.Action)
+		if err != nil {
+			logger.Error(err)
```

Comment:
gitops error needs to also be set, like in here https://github.com/replicatedhq/kots/pull/1702/files#diff-43f6a5e0d4ac97148500c550a148da148479ad4ee1a4e9c5809ea84174e4aa2bR150